db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -19,7 +19,6 @@ def up
       ExtManagementSystem
         .where(:type => ["ManageIQ::Providers::Kubernetes::ContainerManager",
                          "ManageIQ::Providers::Openshift::ContainerManager"])
-        .includes(:custom_attributes).where(:custom_attributes => {:section => "cluster_settings"})
```

Comment:
I wonder why this line causes this error: `PG::UndefinedColumn: ERROR:  column ext_management_systems.security_protocol does not exist` , Can anyone advise? cc @lpichler